### ISSUE TITLE
Landing-pg hero bg flashes blue on load

### ISSUE THREAD

**Issue by ericronne** [18F/calc#1960]
The hero area of the landing page briefly flashes `$color-blue-darkest` as it loads the large background svg …

![rzsjlwmdad](https://user-images.githubusercontent.com/9259185/40547656-1d310150-5ff8-11e8-9eea-cbd331386b75.gif)

**Most of the svg is `#0C4C6F`. Change the color to match that.**

**But hey, we don't have a variable for that color. So in the name of all that is Good and SASSy, let's …**
- [x] Change `$color-blue-darkest` to `#0A2D39`
- [x] Change `$color-blue-darker` to `#0C4C6F`
- [x] Add `$color-blue-dark` as a new variable, and color it `#136B94`
- [x] Swap out instances globally
- [x] Update styleguide

**The new blue set**
In a nutshell, we're adding a new blue and shifting over two existing ones.
![image](https://user-images.githubusercontent.com/9259185/40548598-c2afff62-5ffa-11e8-9748-7e0d261266fd.png)
